instrumentation/aws-sdk/aws-sdk-2.2/library/src/test/java/io/opentelemetry/instrumentation/awssdk/v2_2/DirectLambdaTest.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.awssdk.v2_2;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.propagation.W3CTraceContextPropagator;
+import io.opentelemetry.context.Context;
+import io.opentelemetry.context.propagation.ContextPropagators;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import software.amazon.awssdk.services.lambda.model.InvokeRequest;
+
+public class DirectLambdaTest {
+  private Context context;
+
+  @Before
+  public void setup() {
+    GlobalOpenTelemetry.resetForTest();
+    ;
+    OpenTelemetrySdk.builder()
```

Comment:
I think you don't need to bother setting up the otel sdk yourself. Could just add
```java
  @RegisterExtension
  private static final LibraryInstrumentationExtension testing =
      LibraryInstrumentationExtension.create();
```
that will take care of setting up the sdk. This is what we usually use unless we really need to set up the sdk in some custom way. With this all you really need is
```java
  @Before
  public void setup() {
    testing.runWithHttpServerSpan(() -> { context = Context.current(); });
  }
```

Reply:
Thanks for all the feedback; I've incorporated everything so far.  This in particular was what I needed!  I was wondering why I was struggling so much to get a basic `Context` set up for testing purposes.